DevEdu.API/DevEdu.Business.Tests/AuthenticationServiceTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using DevEdu.Business.Services;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DevEdu.Business.Tests
+{
+    class AuthenticationServiceTests
+    {
+        private AuthenticationService _auth;
```

Comment:
please rename it to _sut to keep it consistent

Reply:
+